packages/@okta/vuepress-site/docs/guides/oauth-for-okta/create-oauth-app/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Create an OAuth 2.0 app in Okta
+---
+Create the client application that you want to use with the Okta APIs.
+
+1. Sign in to your Okta organization as a user with administrative privileges. Don't have one? [Create one for free](https://developer.okta.com/signup).
```

Comment:
Is there a specific admin role that the user needs to use OAuth for Okta APIs? Also, it wasn't completely clear to me if “Create one for free” was for creating a user with admin privileges or for creating an Okta account. For added clarification, maybe change it to "create [an account or org, whichever is the correct terminology] for free"